### PR TITLE
feat(sort-imports): max line length

### PR DESCRIPTION
**Pull request by tthornton3-chwy**
### Description
Closes https://github.com/azat-io/eslint-plugin-perfectionist/issues/83 ! 

Gives support for sorting imports when they are split over multiple lines, thanks to prettier, for example.

### Additional context

If you have any questions or concerns, let me know! I made sure coverage remained at 100%! I wish testing JSON Schemas were easier / part of the plugin, but not yet :(

---

### What is the purpose of this pull request?

- [ ] Bug fix
- [X] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [X] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [X] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [X] Ideally, include relevant tests that fail without this PR but pass with it.
- [X] Read [contribution guidelines](https://github.com/azat-io/eslint-plugin-perfectionist/blob/main/contributing.md).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +12,7 @@ dist/
 # VitePress
 docs/.vitepress/cache/
 docs/.vitepress/temp/
+.vitepress/
```
Comment (azat-io) [inline .gitignore:15]: Why?

Comment (tthornton3-chwy) [reply .gitignore]: I think my IDE or some process put this here-- there was a .vitepress/ folder at the root, with only a `caches` or something like that folder inside. To minimize changes here i just deleted that folder, removed this line so should be all set!

File: test/utils/areOptionsValid.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import type { JSONSchema4 } from '@typescript-eslint/utils/json-schema'
```
[the remark this answers is not in the document]
Comment (tthornton3-chwy) [reply test/utils/areOptionsValid.ts]: Deal! Testing json schema will _eventually_ be a part of the eslint plugin, but for now I just "borrowed" their example method from how their internal team does it itself for now, with the appropriate changes to work for us, which is why the name. But I have now renamed to match this repos style, so should be set!

**Comment by tthornton3-chwy** [comment]
Feedback should be addressed!

**Comment by azat-io** [comment]
Sorry for long response.

Thank you! I'll create a new release soon

**Comment by azat-io** [comment]
Released in [v2.3.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.3.0).

**Comment by tthornton3-chwy** [comment]
Awesome, thanks for looking at it and getting that in and out @azat-io! Have a good week, and thanks for providing this for free and for everyone.

**Pull request merged**